### ISSUE TITLE
[Problem/Bug]: after using put_ZoomFactor, the shortcut key Ctrl+'0' (zoom reset) don't work anymore

### ISSUE THREAD

**Issue by jls28** [MicrosoftEdge/WebView2Feedback#4760]
### What happened?

after using put_ZoomFactor, the shortcut key Ctrl+'0' (zoom reset) don't work anymore

### Importance

Moderate. My app's user experience is affected, but still usable.

### Runtime Channel

Stable release (WebView2 Runtime)

### Runtime Version

126.0.2592.113 or 127

### SDK Version

1.0.2365.46

### Framework

Win32

### Operating System

Windows 10

### OS Version

10.0.19045

### Repro steps

use put_ZoomFactor() with a factor != 1, then try the shortcut Ctrl+'0'

how I found : please read https://github.com/MicrosoftEdge/WebView2Feedback/discussions/4759

with Webview2APISample, 
click on the opened page to give focus,
the shortcuts Ctrl+'+', Ctrl+'-', Ctrl+'0' works.
try the reload button : zoom come back to 100%.
goto menu View/WebView Zoom/0.5x
(now the zoom is 0.5)
then try the shortcut Ctrl+'0' : nothing happen.

In other words, each time put_ZoomFactor is used, this disable the shortcut Ctrl+'0' (zoom reset).

### Repros in Edge Browser

No, issue does not reproduce in the corresponding Edge version

### Regression

No, this never worked

**Comment by victorhuangwq** [comment]
This is an intended behavior

Try this:
- Set View/WebView Zoom/0.5x
- Ctrl + '+', you now zoom out
- Ctrl + '0', you now reset to 0.5x 

 `put_Zoomfactor` sets the new default zoom as `0.5x`, so when you do Ctrl + `0` it now resets to 0.5x, thus in your steps nothing happened. But if you Ctrl +, and then do Ctrl 0, it returns back to `0.5x`.
 
 https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2controller.zoomfactor?view=webview2-dotnet-1.0.2651.64#remarks
